common/src/importers/keeperImporters/keeperJsonImporter.ts

```diff
@@ -0,0 +1,68 @@
+import { BaseImporter } from ".././baseImporter";
+import { Importer } from ".././importer";
+
+import { ImportResult } from "../../models/domain/importResult";
+
+import { KeeperJsonExport, RecordsEntity } from "./types/keeperJsonTypes";
+
+export class KeeperJsonImporter extends BaseImporter implements Importer {
+  parse(data: string): Promise<ImportResult> {
+    const result = new ImportResult();
+    const keeperExport: KeeperJsonExport = JSON.parse(data);
+    if (keeperExport == null || keeperExport.records == null || keeperExport.records.length === 0) {
+      result.success = false;
+      return Promise.resolve(result);
+    }
+
+    keeperExport.records.forEach((record) => {
+      this.parseFolders(result, record);
+
+      const cipher = this.initLoginCipher();
+      cipher.name = record.title;
+      cipher.login.username = record.login;
+      cipher.login.password = record.password;
+
+      cipher.login.uris = this.makeUriArray(record.login_url);
+      cipher.notes = record.notes;
+
+      let customfieldKeys = Object.keys(record.custom_fields);
+      if (customfieldKeys.includes("TFC:Keeper") && record.custom_fields["TFC:Keeper"] != null) {
```

Comment:
The first part of this condition seems unnecessary, if there's no key then `record.custom_fields["TFC:Keeper"]` will always return `null` anyway.

Reply:
That is true, removed the unnecessary check for the key